services/sourceforge/sourceforge-languages.service.js

```diff
@@ -0,0 +1,51 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  categories: Joi.object({
+    language: Joi.array().required(),
+  }).required(),
+})
+
+export default class SourceforgeLanguages extends BaseJsonService {
+  static category = 'analysis'
+
+  static route = {
+    base: 'sourceforge/languages',
+    pattern: ':project',
+  }
+
+  static examples = [
+    {
+      title: 'SourceForge languages',
+      namedParams: {
+        project: 'mingw',
+      },
+      staticPreview: this.render(6),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'languages' }
+
+  static render(languages) {
+    return {
+      message: languages,
```

Comment:
I suspect we're less likely to hit a project where it makes a difference, but lets format this with `metric()` too, especially given we're expecting `isMetric` in the test